### PR TITLE
Zeus modules

### PR DESCRIPTION
**Pull request by AtrixZockt**
This PR adds Zeus Modules for KAT to the game.
Currently usable modules:

- Manage Units Airways
  ° Give / Take Pneumothorax from unit
  ° Give / Take Tensionpneumothorax from unit
  ° Give / Take Hemopneumothorax
  ° Give / Take Airway obstruction
  ° Give / Take Airway occlusion
  ° Set the SpO2 value for the unit

- Change blood group / volume
  ° Give unit another blood group
  ° Set the blood volume for the unit (0-6L)

- Change shockable state
  ° Change the shockable state for the unit (from KAT-Pharmacy)

- Open Medical menu
  ° Opens the medical menu for the selectet unit so you can inspect them from the distance 
   (You can only see it but can not bandage the unit)

This is all tested and worked without any errors

### CODE REVIEW

**Comment by YetheSamartaka** [comment]
Part of this PR are also some stringtable fixes for other parts of KAT(Italian)

**Comment by MiszczuZPolski** [comment]
> Part of this PR are also some stringtable fixes for other parts of KAT(Italian)

And Korean for pharma

File: addons/zeus/functions/fnc_ui_changeBloodType.sqf [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+};
+
+
+private _fnc_sliderMove = {
+    params ["_slider","_curBloodVol"];
+    private _idc = ctrlIDC _slider;
+	private _logic = GETMVAR(BIS_fnc_initCuratorAttributes_target,objNull);
+	private _unit = attachedTo _logic;
+    private _curVal = _unit getVariable ["ace_medical_bloodVolume", 6.0];
+    _slider ctrlSetTooltip format ["%1%3 (was %2%3)", parseNumber((sliderPosition _slider) toFixed 2), (parseNumber (_curVal toFixed 2)), "L"];
```
Comment (Tomcat-SG) [inline addons/zeus/functions/fnc_ui_changeBloodType.sqf]: Tooltip should be added to the stringtable.xml too

File: addons/zeus/functions/fnc_ui_manageAirway.sqf [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    deleteVehicle _logic;
+};
+
+private _fnc_sliderMove = {
+    params ["_slider","_curSpO2Val"];
+    private _idc = ctrlIDC _slider;
+	private _logic = GETMVAR(BIS_fnc_initCuratorAttributes_target,objNull);
+	private _unit = attachedTo _logic;
+    private _curVal = _unit getVariable ["KAT_medical_airwayStatus", 50];
+    _slider ctrlSetTooltip format ["%1%3 (was %2%3)", round(sliderPosition _slider), round _curVal, "%"];
```
Comment (Tomcat-SG) [inline addons/zeus/functions/fnc_ui_manageAirway.sqf]: Tooltip should be added to the stringtable.xml too

File: addons/zeus/functions/fnc_ui_manageAirway.sqf [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+_display displayAddEventHandler ["unload", _fnc_onUnload];
+_ctrlButtonOK ctrlAddEventHandler ["buttonclick", _fnc_onConfirm];
+
+
+
+
+
+
+
```
Comment (Tomcat-SG) [inline addons/zeus/functions/fnc_ui_manageAirway.sqf]:
```suggestion
```

File: addons/zeus/functions/fnc_ui_manageAirway.sqf [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+	_unit setVariable [QEGVAR(airway,obstruction), _valueArr select 0, true];
+	_unit setVariable ["KAT_medical_airwayOccluded", _valueArr select 1, true];
+	_unit setVariable ["KAT_medical_pneumothorax", _valueArr select 2, true];
+	_unit setVariable ["KAT_medical_hemopneumothorax", _valueArr select 3, true];
+	_unit setVariable ["KAT_medical_tensionpneumothorax", _valueArr select 4, true];
+	private _curSpO2Val = _unit getVariable ["KAT_medical_airwayStatus", 50];
+	private _sliderValue = sliderPosition (_display displayCtrl 16106);
+    
+	_unit setVariable ["KAT_medical_airwayStatus", round(_sliderValue), true]; 
+    if(_curSpO2Val == 100) then {
```
Comment (Tomcat-SG) [inline addons/zeus/functions/fnc_ui_manageAirway.sqf]:
```suggestion
    if(_curSpO2Val isEqualTo 100) then { 
```

File: optionals/usaf/config.cpp [hunk 5]
```diff
@@ -4,12 +4,12 @@ class CfgPatches {
...
-        units[] = {};
-        weapons[] = {};
+        units[] = { };
+        weapons[] = { };
         magazines[] = {};
         requiredAddons[] = {
-            "kat_misc",
-            "rhsusf_main"
+            "kat_acemisc",
+            "rhs_usaf"
```
Comment (Tomcat-SG) [inline optionals/usaf/config.cpp]: these are the old names? why should we change this to a older name we dont use anymore?

**Review by Tomcat-SG — CHANGES_REQUESTED** [review]
Change these little things and we are ready to merge 👍🏼

File: addons/zeus/functions/fnc_ui_changeBloodType.sqf (hunk 1 above)
Comment (YetheSamartaka) [reply addons/zeus/functions/fnc_ui_changeBloodType.sqf]: Solved

File: addons/zeus/functions/fnc_ui_manageAirway.sqf (hunk 2 above)
Comment (YetheSamartaka) [reply addons/zeus/functions/fnc_ui_manageAirway.sqf]: Solved

File: addons/zeus/functions/fnc_ui_changeBloodType.sqf [hunk 6]
```diff
@@ -57,7 +57,7 @@ private _fnc_sliderMove = {
 	private _logic = GETMVAR(BIS_fnc_initCuratorAttributes_target,objNull);
 	private _unit = attachedTo _logic;
     private _curVal = _unit getVariable ["ace_medical_bloodVolume", 6.0];
-    _slider ctrlSetTooltip format ["%1%3 (was %2%3)", parseNumber((sliderPosition _slider) toFixed 2), (parseNumber (_curVal toFixed 2)), "L"];
+    _slider ctrlSetTooltip format ["STR_kat_zeus_sliderFormat13was23", parseNumber((sliderPosition _slider) toFixed 2), (parseNumber (_curVal toFixed 2)), "L"];
```
Comment (Tomcat-SG) [inline addons/zeus/functions/fnc_ui_changeBloodType.sqf]:
```suggestion
    _slider ctrlSetTooltip format [localize "STR_kat_zeus_sliderFormat13was23", parseNumber((sliderPosition _slider) toFixed 2), (parseNumber (_curVal toFixed 2)), "L"];
```

File: addons/zeus/functions/fnc_ui_manageAirway.sqf [hunk 7]
```diff
@@ -56,7 +56,7 @@ private _fnc_sliderMove = {
 	private _logic = GETMVAR(BIS_fnc_initCuratorAttributes_target,objNull);
 	private _unit = attachedTo _logic;
     private _curVal = _unit getVariable ["KAT_medical_airwayStatus", 50];
-    _slider ctrlSetTooltip format ["%1%3 (was %2%3)", round(sliderPosition _slider), round _curVal, "%"];
+    _slider ctrlSetTooltip format ["STR_kat_zeus_sliderFormat13was23", round(sliderPosition _slider), round _curVal, "%"];
```
Comment (Tomcat-SG) [inline addons/zeus/functions/fnc_ui_manageAirway.sqf]:
```suggestion
    _slider ctrlSetTooltip format [localize "STR_kat_zeus_sliderFormat13was23", round(sliderPosition _slider), round _curVal, "%"];
```

**Review by Tomcat-SG — CHANGES_REQUESTED** [review]

**Review by Tomcat-SG — APPROVED** [review]
LGTM

**Pull request merged**